### PR TITLE
Update EnemyCounter.lua

### PR DESCRIPTION
**Pull request by nuttin0011**
CheckInteractDistance(name, 2) is check from center of unit in 8 yard.
IsItemInRange("item:34368", name) is check from hit box of unit in 8 yard.

CheckInteractDistance is error when enemy has large hit box as boss.

### CODE REVIEW

File: modules/EnemyCounter.lua [hunk 1]
```diff
@@ -55,7 +55,7 @@ function TMW_ST:CountInRange(stop)
     for i = 1, EnemyCounter.max_scan do
         name = 'nameplate' .. i
 
-        if _UnitExists(name) and _UnitReaction(name, "player") < 5 and _CheckInteractDistance(name, 2) == true then
+        if _UnitExists(name) and _UnitReaction(name, "player") < 5 and IsItemInRange("item:34368", name) == true then
```
Comment (smira2000) [inline modules/EnemyCounter.lua:58]: Hi @nuttin0011 what item is that?
Just thought I'd ask :)

Comment (arieh) [inline modules/EnemyCounter.lua:58]: can you share where you took this from? I understand what you're trying to fix but I'm always hesitant of adding things like this (eg - what's this item and why is it a better indicator for melee range)?

Comment (nuttin0011) [reply modules/EnemyCounter.lua:58]: it's from LibRangeCheck-2.0
https://github.com/WeakAuras/LibRangeCheck-2.0/blob/7d716f7264e463a05471c00a1c36f3cdadaadbe8/LibRangeCheck-2.0/LibRangeCheck-2.0.lua#L321

Comment (arieh) [reply modules/EnemyCounter.lua:58]: I think a better solution then would just be to use LibRangeCheck directly. It's included in TellMeWhen core anyway so it's safe to assume it's present, and that way if the lib updates we get it for free

**Comment by arieh** [comment]
I've changed EnemyCounter to use LibRangeCheck. Thanks for pointing out the issue!

**Pull request closed without merging**